src/graphics/camera_normal.cpp

```diff
@@ -68,8 +68,8 @@ CameraNormal::CameraNormal(Camera::CameraType type,  int camera_index,
 //-----------------------------------------------------------------------------
 /** Moves the camera smoothly from the current camera position (and target)
  *  to the new position and target.
- *  \param wanted_position The position the camera wanted to reach.
- *  \param wanted_target The point the camera wants to point to.
+ *  \param dt Delta time, 
+ *  It's expected in SnapToPosition() that setting DT to 1 snaps to the destination
```

Comment:
that assumption seems weird to me, "dt" is the number of seconds elapsed since last frame. So 1 means "1 second", not necessarily "snap to end".  Calling the code with a large dt will simulate that a lot of time happened and skip the animation, so the logic is okay, the comment isn't quite true though